layer.md

```diff
@@ -44,10 +44,46 @@ Where supported, MUST include file attributes for Additions and Modifications in
     * Group Name (`gname`) *secondary to `gid`*
 * Mode (`mode`)
 * Extended Attributes (`xattrs`)
-* Symlink reference (`linkname`)
+* Symlink reference (`linkname` + symbolic link type)
+* [Hardlink](#hardlinks) reference (`linkname`)
 
 [Sparse files](https://en.wikipedia.org/wiki/Sparse_file) SHOULD NOT be used because they lack consistent support across tar implementations.
 
+#### Hardlinks
+
+Hardlinks are a [POSIX concept](http://pubs.opengroup.org/onlinepubs/9699919799/functions/link.html) for having one or more directory entries for the same file on the same device.
+Not all filesystems support hardlinks (e.g. [FAT](https://en.wikipedia.org/wiki/File_Allocation_Table)).
+
+Hardlinks are possible with all [file types](#file-types) except `directories`.
+Non-directory files have hardlinks when the linkcount is greater than 1.
+Hardlinked files are on a same device (i.e. comparing Major:Minor pair), and have the same inode.
+The corresponding files that share the link with the > 1 linkcount may be outside the directory that the changeset is being produced from, in which case the `linkname` is not recorded in the changeset.
+
+Hardlinks are stored in a tar archive with type of '`1`', per the [GNU Basic Tar Format][gnu-tar-standard] and [libarchive(5)][libarchive].
```

Comment:
Also, the page you are linking is libarchive's tar(5), not libarchive(5).  I think you want a line like:

    Hardlinks are stored in a tar archive with type of `1` as specified by [IEEE Std 1003.1-1988][ieee1003.1-1998] (“ustar”, discussed [here][gnu-tar-standard] and [here][tar.5-ustar]).

    [ieee1003-1998]: http://standards.ieee.org/findstds/standard/1003.1-1988.html
    [gnu-tar-standard]: http://www.gnu.org/software/tar/manual/html_node/Standard.html
    [tar.5-ustar]: https://github.com/libarchive/libarchive/wiki/ManPageTar5#POSIX_ustar_Archives